Makefile

```diff
@@ -284,6 +284,33 @@ test-cluster:
 get-cluster-credentials: set-azure-account
 	az aks get-credentials --overwrite-existing -g ${CLUSTER_RESOURCE_GROUP_NAME} -n ${CLUSTER_NAME}
 
+define get_app_first_pod
+  $(if $(env), , $(error Missing <env>. Usage: "make <env> command"))
+	$(eval POD_NAME=$(shell kubectl -n $(AZURE_NAMESPACE) get pods | grep -- teaching-vacancies-$(env)-[0-9] | awk '{print $$1}'))
+endef
+
+# make review pr_id=5432 ssh
+# make qa ssh
+.PHONY: ssh
+ssh:
```

Comment:
It's not really ssh though. Maybe call it `shell:` ?

Reply:
So much better. Thanks!